integration/scenarios/export_service.go

```diff
@@ -40,10 +40,18 @@ func NewExportServiceScenario(cfg *aws.Config, nsName string, svcName string, po
 
 	for _, ip := range strings.Split(ips, ",") {
 		endpts = append(endpts, &model.Endpoint{
-			Id:         model.EndpointIdFromIPAddress(ip),
-			IP:         ip,
-			Port:       int32(port),
-			Attributes: make(map[string]string, 0),
+			Id: model.EndpointIdFromIPAddress(ip, int32(port)),
+			IP: ip,
+			ServicePort: model.Port{
+				Port:       int32(port),
```

Comment:
This is causing integration test test to fail for me, since service port is 8080 and regular port is 80. We need to extract and pass both into this test.
You can run `make integration-suite` locally to verify this.